lm_eval/tasks/superglue.py

```diff
@@ -324,6 +324,40 @@ def has_test_docs(self):
     def test_docs(self):
         return self.dataset["test"]
 
+    def process_results(self, doc, results):
+        """Take a single document and the LM results and evaluates, returning a
+        dict where keys are the names of submetrics and values are the values of
+        the metric for that one document
+
+        :param doc:
+            The document as returned from training_docs, validation_docs, or test_docs.
+        :param results:
+            The results of the requests created in construct_requests.
+        """
+        target = self.doc_to_target(doc).strip()
+
+        answer_choices_list = self.prompt.get_answer_choices_list(doc)
+        pred = answer_choices_list[np.argmax(results)]
+
+        out = {
+            "parity": (doc["idx"], pred),
+            "acc": pred == target
+        }
+
+        if self.save_examples:
+            example = {"doc": doc, "pred": pred}
+            return out, example
+        return out
+
```

Comment:
Could you add the fields to the logging example? Probably don't need to add `doc`?
```
                "target": target,
                "answer_choices_list": answer_choices_list,
```

Reply:
Done in the next commit.